ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -292,16 +303,23 @@ open class BadgeView: UIView {
             accessibilityHint = "Accessibility.Select.Hint".localized
         }
     }
+    
+    private func update() {
+        updateBackgroundColor()
+        updateLabelTextColor()
+    }
 
     private func updateBackgroundColor() {
         if let window = window {
-            backgroundView.backgroundColor = backgroundColor(for: window, style: style, selected: isSelected, enabled: isActive)
+            let color = backgroundColor ?? backgroundColor(for: window, style: style, selected: isSelected, enabled: isActive)
```

Comment:
Can we assign it directly to the background color since the color constant is not being used anywhere else? (just like you did in the text color)